canonical/charmhub.io
# fix: Revert beta store cleanup

## Done
Reverted the changes made to clean up the beta store as it has caused some problems such as broken redirects

## How to QA
- Go to https://charmhub-io-1967.demos.haus/
- Click around and make sure things work as expected
